.env.example

```
@@ -1,13 +1,50 @@
 # DB
-DB_URL=jdbc:postgresql://db:5432/userdb   # docker-compose 서비스명 사용
 DB_USERNAME=postgres
 DB_PASSWORD=your_password
 
+# 로컬 실행용(로컬, 배포용 설정 중 하나는 주석 처리할 것)
+DB_URL=jdbc:postgresql://localhost:5432/userdb
+
+# docker 기반 배포용
+DB_URL=jdbc:postgresql://db:5432/userdb
```

dotenv-linter (4.0.0)

[warning] [DuplicatedKey] The DB_URL key is duplicated

(DuplicatedKey)